features/steps/population_types_component.go

```diff
@@ -4,6 +4,9 @@ import (
 	"context"
 	"net/http"
 
+	"github.com/ONSdigital/dp-api-clients-go/v2/cantabular"
+	dperrors "github.com/ONSdigital/dp-api-clients-go/v2/errors"
+	"github.com/ONSdigital/log.go/v2/log"
 	"github.com/pkg/errors"
```

Comment:
These imports can probably be sorted better (we usually separate ONS and 3rd party libs). There are a couple of other files below this can be applied to too.

Reply:
This has been addressed.